sourmash/lca/lca_db.py

```diff
@@ -169,8 +168,21 @@ def insert(self, sig, ident=None, lineage=None):
         # for hashval in minhash.get_mins():
         #     self.hashval_to_idx[hashval].add(idx)
 
-        return self._methodcall(lib.LcaDB_insert, sig._objptr, to_bytes(ident), list(lineage), len(list(lineage)))
-        # return len(minhash)
+        # trying to put lineage into a json
+        lineage_dict = OrderedDict()
+        if lineage:
+            for pair in lineage:
+                lineage_dict[pair[0]] = pair[1]
+        else:
+            lineage = ""
```

Comment:
You can use `dict(lineage)` here:

```suggestion
        if lineage:
            lineage_dict = dict(lineage)
        else:
            lineage = ""
```